packages/backend-common/package.json

```diff
@@ -81,6 +82,7 @@
   },
   "devDependencies": {
     "@backstage/cli": "^0.9.0",
+    "@backstage/core-app-api": "^0.1.21",
```

Comment:
Nope :grin: import `ConfigReader` from `@backstage/config` instead

Reply:
:+1: done in 42b01ee725